restlight-server/src/test/java/esa/restlight/server/config/ServerOptionsTest.java

```diff
@@ -49,16 +56,17 @@ void testConfigure() {
                 .route(null)
                 .ssl(null)
                 .https(null)
+                .bizThreads(bizThreads)
                 .configured();
 
         assertTrue(options.isHttp2Enable());
         assertTrue(options.isUseNativeTransports());
         assertEquals(2, options.getConnectorThreads());
         assertEquals(3, options.getIoThreads());
-        assertEquals(4, options.getBizThreadsCore());
-        assertEquals(5, options.getBizThreadsMax());
-        assertEquals(6, options.getBizThreadsBlockingQueueLength());
-        assertEquals(7, options.getBizThreadsKeepAliveTimeSeconds());
```

Comment:
plz do not remove existing unit tests of deprecated functions

Reply:
thx, have reverted!